src/Symfony/Component/Workflow/Event/Event.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\Event;
+
+use Symfony\Component\EventDispatcher\Event as BaseEvent;
+use Symfony\Component\Workflow\Marking;
+use Symfony\Component\Workflow\Transition;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+class Event extends BaseEvent
+{
+    private $subject;
+    private $marking;
+    private $transitions;
+
+    public function __construct($subject, Marking $marking, Transition $transitions)
+    {
+        $this->subject = $subject;
+        $this->marking = $marking;
+        $this->transitions = $transitions;
+    }
+
+    public function getMarking()
+    {
+        return $this->marking;
+    }
+
+    public function getSubject()
+    {
+        return $this->subject;
+    }
+
+    public function getTransition()
```

Comment:
Thanks not noting me this. The issue was on the attributes. So it's  `getTransition` and I renamed the attributes from `transitions` to `transition`.